prboom2/src/gl_preprocess.c

```diff
@@ -695,6 +706,78 @@ static void gld_PrecalculateSector(int num)
   Z_Free(lineadded);
 }
 
+static dboolean gld_SectorIsSelfReferencing(sector_t* sector)
+{
+  int i;
+
+  for (i = 0; i < sector->linecount; ++i) {
```

Comment:
I realize it's a bit annoying because the style is different in different files, but please put the `{` below the line here (and in the other cases).